src/Dafny/examples/fs/indirect_fs.dfy

```diff
@@ -196,12 +196,12 @@ module IndFs
     constructor Init(d: Disk)
       ensures ValidQ()
       ensures fresh(Repr)
-      ensures data == imap pos: Pos {:trigger pos.idx.data?} | pos.idx.data? :: block0
+      ensures data == imap pos: Pos {:trigger pos.idx.data?()} | pos.idx.data?() :: block0
```

Comment:
ah, actually, now that I read this diff again, this might have been the trigger that I found at the boogie level. 

@RustanLeino I'm not sure triggering on a pattern ending in a constant should be allowed, since constants are inline functions at the Boogie level.

Reply:
except, wait... if this trigger was the problem I don't see how it would have global effect, since it's in the postcondition of `Init`. hmm...